paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentOptionEditState.kt

```diff
@@ -2,6 +2,5 @@ package com.stripe.android.paymentsheet.ui
 
 internal enum class PaymentOptionEditState {
     None,
-    Removable,
```

Comment:
Should we follow up to make this a boolean?

Reply:
yes sgtm!! Adding to my ticket for clean up ideas